fixtures/steel-pro-technologies/steel-pro.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "steel pro",
+  "shortName": "40x3 par led",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["EJGR"],
+    "createDate": "2023-10-27",
+    "lastModifyDate": "2023-10-27"
+  },
+  "links": {
+    "manual": [
+      "https://lite-tek.com.mx/manuales/PAR-18X10.pdf"
```

Comment:
This URL is a manual for a "LITETEK PAR 18x10", which looks completely different from the "steel pro 40x3 par led".

I did find a "STEELPRO TECHNOLOGIES" at https://steelprotechnologies.wixsite.com/steelprotechnologies/blank but their PARs have a different naming system ("EXACT-..."), and I don't see any LED PARs that have 40 x 3 of anything.

I did find this picture of a manual for a "403-MPR" which could be it: https://forums.pioneerdj.com/hc/en-us/community/posts/11206229361049-Steelpro-403-MPR